README.md

```diff
@@ -194,6 +196,21 @@ This uses `attn_order=12` which includes `prime_attention` layers with keys/valu
 
 If you instead want to use a model with the usual encoder-decoder style transformer, use `small_sep_enc_dec_prior`.
 
+### Fine-tune pre-trained top-level prior to new style(s)
+Previously, we showed how to train a small top-level prior from scratch. Assuming you have a GPU with at least 15 GB of memory and support for fp16, you could fine-tune from our pre-trained 1B top-level prior. Here are the steps:
+
+1. Support `--labels=True` by implementing `get_metadata` in `jukebox/data/files_dataset.py` for your dataset.
+2. Add new entries in `jukebox/data/ids`. 
+  * We recommend replacing existing mappings (e.g. rename `"unknown"`, etc with styles of your choice; you will have to create a new branch for this). This uses the pre-trained style vectors as initialization and could potentially save some compute.
+3. Run
+```
+mpiexec -n {ngpus} python jukebox/train.py --hps=vqvae,prior_1b_lyrics,all_fp16,cpu_ema --name=finetuned \
+--sample_length=1048576 --bs=1 --aug_shift --aug_blend --audio_files_dir={audio_files_dir} \
+--labels=True --train --test --prior --levels=3 --level=2 --weight_decay=0.01 --save_iters=1000 \
+--min_duration=17.84 --max_duration=600 --c_res=1 --lr_use_linear_decay --lr_decay={decay_steps_as_needed}
+```
```

Comment:
I followed the convention to change these hps directly in the dictionary in hparams.py so that they are correctly used when we call make_models.py for sampling. You might want to do same here?

Reply:
sg! some default hps values are moved to the dictionary now

Reply:
Same for linear decay here as above?